### PR TITLE
TR-1677 fixing logic in detection to avoid null pointer exception errors

### PR DESCRIPTION
**Pull request by josehelps**
# PR Template for new Detections

For Authors:
1. Make sure that CI/CD [detection-testing and build-and-validate](https://github.com/splunk/security_content/actions) jobs passed ✔️. 

For Reviewers:
- [ ] Verify CI/CD jobs have passed without errors.
- [x] Validate SPL logic.
- [x] Validate tags, description, and how to implement.
- [ ] Validate name patches `<platform>_<mitre att&ck technique>_<short description>`

### CODE REVIEW

File: detections/endpoint/ssa___detect_dump_lsass_memory_using_comsvcs.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ search: '| from read_ssa_enriched_events() | eval tenant=ucast(map_get(input_eve
   "string", null), process_name=lower(ucast(map_get(input_event, "process_name"),
   "string", null)), timestamp=parse_long(ucast(map_get(input_event, "_time"), "string",
   null)), process=lower(ucast(map_get(input_event, "process"), "string", null)), event_id=ucast(map_get(input_event,
-  "event_id"), "string", null) | where process_name LIKE "%rundll32.exe%" AND match_regex(process,
+  "event_id"), "string", null) | where cmd_line IS NOT NULL AND process_name LIKE "%rundll32.exe%" AND match_regex(process,
```
Comment (ljstella) [inline detections/endpoint/ssa___detect_dump_lsass_memory_using_comsvcs.yml]: Do we need to be null-checking `cmd_line` in this detection, or should we be checking for nulls in `process_name` before we use a LIKE function on it?

Comment (josehelps) [reply detections/endpoint/ssa___detect_dump_lsass_memory_using_comsvcs.yml]: we prob should let me update the detection

Comment (josehelps) [reply detections/endpoint/ssa___detect_dump_lsass_memory_using_comsvcs.yml]: fixed

**Review by ljstella — DISMISSED** [review]
Think that will work.

**Comment by josehelps** [comment]
Passed Testing ✔️ 

* ssa___wbadmin_delete_system_backups.yml

```
2022-02-04 16:23:47,827 INFO - PASSED ssa___wbadmin_delete_system_backups.yml anomalies=2 decorated_events=8
2022-02-04 16:23:47,828 INFO - overall status=SUCCESS detections=1 ok=1 failed=0 errors=0
2022-02-04 16:23:47,828 INFO - details written to /tmp/detection_check/71efbf52-4dbb-4c00-a520-306aa546cbb7/anomalies.json
```
* ssa___detect_dump_lsass_memory_using_comsvcs.yml

```
2022-02-04 16:08:49,485 INFO - MATCHED 76bb9e35-f314-4c3d-a385-83c72a13ce4e anomaly_count=2 event_count=4 file=/Users/jhernandez/splunk/security_content/dist/ssa/srs/ssa___detect_dump_lsass_memory_using_comsvcs.yml
2022-02-04 16:08:49,488 INFO - PASSED ssa___detect_dump_lsass_memory_using_comsvcs.yml anomalies=2 decorated_events=4
2022-02-04 16:08:49,488 INFO - overall status=SUCCESS detections=1 ok=1 failed=0 errors=0
2022-02-04 16:08:49,488 INFO - details written to /tmp/detection_check/76bb9e35-f314-4c3d-a385-83c72a13ce4e/anomalies.json
```

**Review by ljstella — APPROVED** [review]
LGTM

**Pull request merged**